browser-extension/src/models/tournesolVideosBox/TournesolVideosBox.js

```diff
@@ -0,0 +1,219 @@
+import { convertDurationToClockDuration } from '../../utils.js';
+
+export class TournesolVideosBox {
```

Comment:
Singular if this class represents the container of a single video.

```suggestion
export class TournesolVideoBox {
```

(minor: we can even use the same vocabulary as in the front end `TournesolVideoCard`)

Reply:
solved by [04c4b77](https://github.com/tournesol-app/tournesol/pull/1436/commits/04c4b77d4bdad156925140fbed04ba0f6097365a)